sdk/core/Azure.Core/src/Pipeline/HttpPipelineBuilder.cs

```diff
@@ -13,10 +13,12 @@ namespace Azure.Core.Pipeline
     /// </summary>
     public static class HttpPipelineBuilder
     {
+        private static int DefaultPolicyCount = 8;
```

Comment:
Added in response to https://github.com/Azure/azure-sdk-for-net/pull/41773/files#r1492660773